src/elm/Config.elm

```diff
@@ -4,12 +4,12 @@ accessToken : String
 accessToken = "SujuZGJMeSX2yRwcqRpXX6RtF2LJPY6Q2QBXovA-5Uc"
 
 backendUrl : String
-backendUrl = "http://192.168.1.27/productivity/www"
+backendUrl = "http://live-productivity.pantheon.io/"
```

Comment:
in short in the repo we should probably have `src/elm/Config.elm.example` and have `src/elm/Config.elm` in .gitignore